qualtran/bloqs/chemistry/trotter/hubbard/qpe_cost_optimization.ipynb

```diff
@@ -446,6 +446,59 @@
     "print(rf\"T_{{opt}} = {t_opt:4.3e}\")"
    ]
   },
+  {
+   "cell_type": "markdown",
```

Comment:
is this notebook unit tested anywhere? please add and make sure to `@pytest.mark.notebook`

Reply:
For some reason I thought the scipts automagically globbed all the notebooks?

Reply:
the CI will automatically git-glob all the notebooks in the notebooks check, but it's policy :tm: to also include a unit test that will get picked up by e.g. a local launch of `pytest`

Reply:
done